### PR TITLE
Adding new thanksgiving post.

### PR DESCRIPTION
**Pull request by rdegges**
Adding a new Thanksgiving article. Please merge asap =)

### CODE REVIEW

File: _source/_posts/2017-11-22-three-developer-tools-im-thankful-for.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+on something a little more in line with the theme of this website: my favorite
+developer tools of all time.
+
+These are the tools I use every single day, and have been using for many years.
+A lot of the time I take them for granted since I use them constantly, but
+they’re all essential tools in my development arsenal, and I want to take a
+minute to share just why I’m so thankful for them.
+
+
+## vim
```
Comment (mraible) [inline _source/_posts/2017-11-22-three-developer-tools-im-thankful-for.md]: https://vim.sourceforge.io/ seems to call it "Vim" with a capital V.

File: _source/_posts/2017-11-22-three-developer-tools-im-thankful-for.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+I could probably write an entire essay about all the amazing people I know and
+friends I have, but I’ll save that for another time. Today I wanted to reflect
+on something a little more in line with the theme of this website: my favorite
+developer tools of all time.
+
+These are the tools I use every single day, and have been using for many years.
+A lot of the time I take them for granted since I use them constantly, but
+they’re all essential tools in my development arsenal, and I want to take a
+minute to share just why I’m so thankful for them.
+
```
Comment (mraible) [inline _source/_posts/2017-11-22-three-developer-tools-im-thankful-for.md]: Doesn't matter when processed, but I prefer to remove extra line breaks.

File: _source/_posts/2017-11-22-three-developer-tools-im-thankful-for.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+Literally everything I do on the command line is powered by the Z shell
+([zsh](http://www.zsh.org/)) and the aptly named [oh-my-zsh](http://ohmyz.sh/)
+project which bootstraps it.
+
+Most developers are used to using Bash as their standard shell, but zsh
+provides some incredibly nice features (especially when paired with oh-my-zsh):
+
+Easy to install plugins A massive community Fast, incremental search Incredible
+customization SPEED!!#@~
```
Comment (mraible) [inline _source/_posts/2017-11-22-three-developer-tools-im-thankful-for.md]: Are the trailing characters a typo, or something I don't understand? ;)

**Comment by rdegges** [comment]
Fixed.

**Pull request merged**